api/server.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const cors = require('cors');
+
+const server = express();
+server.use(cors());
+server.use(express.json());
+
+const postRoutes = require('./routes/posts')
+
+server.use('/posts', postRoutes)
```

Comment:
you use /posts here but you use just "/" in the /routes section so I think theres a clash.